WeatherForecast/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,122 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+
+namespace ConsoleApp
+{
+    public class Program
+    {
+        static async Task Main(string[] args)
+        {
+            //    var builder = new ConfigurationBuilder();
```

Comment:
If you dont use this file, then delete it

Reply:
Fixed versioning problem after upgrade to 6.0 and now its usable.